### DISCUSSION TITLE
No icons for Assembly or Makefiles?

### DISCUSSION THREAD

**Discussion by danmartinvela** [zed-industries/zed#12849] [Feature Requests]
just that, also as a suggestion

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
